src/components/input.js

```diff
@@ -240,6 +240,14 @@ const withAutocomplete = WrappedComponent => ({
     { itemToString: v => v ? itemToString(v) : value } :
     { selectedItem: value }
 
+  const inputEl = useRef()
```

Comment:
In order to use the x to clear the search, we need this. I used a reference that does this in the same file, for "ConfirmedSearchInput"